.github/workflows/run-archiver.yml

```diff
@@ -3,6 +3,22 @@ name: run-archiver
 
 on:
   workflow_dispatch:
+    inputs:
+      small_runner:
```

Comment:
These variables will all only set if there's a manual workflow dispatch. If the action runs as scheduled, these variables should _I believe_ evaluate as empty strings, but the documentation on boolean behavior is sparse.